roles/installer/tasks/enable_metrics_utility.yml

```diff
@@ -0,0 +1,24 @@
+---
+# Check to make sure provided pvc exists, error loudly if not.  Otherwise, the management pod will just stay in pending state forever.
+- name: Check provided PVC claim exists
+  kubernetes.core.k8s_info:
+    name: "{{ _metrics_utility_pvc_claim }}"
+    kind: PersistentVolumeClaim
+    namespace: "{{ ansible_operator_meta.namespace }}"
+  register: provided_pvc
```

Comment:
this `provided_pvc` registered value isn't used elsewhere. Could we removed it ?